RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -35,6 +36,7 @@
                                            #'keep ZDCDataFramesSorted_*Digis_*_*',
                                            'keep ZDCDataFramesSorted_hcalDigis_*_*',
                                            'keep ZDCDataFramesSorted_castorDigis_*_*',
+                                           'keep QIE10DataFrameHcalDataFrameContainer_hcalDigis_ZDC_*',
```

Comment:
Did HI decide to write RECO for this pA run?
We didn't write it for PbPb last year and we do not write it now in pp for more than a half of PDs.

You should edit RecoHI/Configuration/python/customise_PPwithHI.py in line with other ZDC additions to AOD